src/components/PersonInfo/PersonInfo.tsx

```diff
@@ -0,0 +1,23 @@
+import { Person } from '../../types/Person';
+import './PersonInfo.scss';
+
+type Props = {
+  person: Person,
+  onClick: (person: Person) => void,
+};
+
+export const PersonInfo: React.FC<Props> = ({ person, onClick }) => {
+  return (
+    <div
+      className="dropdown-item"
+      onClick={() => onClick(person)}
+      onKeyUp={() => {}}
+      role="menuitem"
```

Comment:
The role 'menuitem' might not be appropriate for a non-interactive element like 'div'. Consider using a more semantic HTML tag (like 'button' or 'a') or manage the keyboard events properly.